backend/src/routes/blog.ts

```diff
@@ -204,13 +204,15 @@ blogRouter.get("/:id", async (c) => {
 
 blogRouter.delete("/:id", async (c) => {
   try {
-		const prisma = getDBInstance(c);
-    const postId = await c.req.param("id");
-    const post = await prisma.post.delete({
-      where: {
-        id: postId,
-      },
-    });
+    const postId = c.req.param("id");
+    const prisma = new PrismaClient({
+      datasourceUrl: c.env.DATABASE_URL,
+    }).$extends(withAccelerate());
+
+    await prisma.post.delete({
+      where: {id: postId}
+    })
+
```

Comment:
This
```bash
const prisma = new PrismaClient({
      datasourceUrl: c.env.DATABASE_URL,
    }).$extends(withAccelerate());
```
 should be replaced with 
```bash
const prisma = getDBInstance(c);
```

Actually, after this change, this entire code-snippet block remains the same as is.